server/server.js

```diff
@@ -17,7 +16,7 @@ module.exports = function(port, googleAuthoriser) {
                 console.log("success");
                 adminToken = token;
                 res.cookie("sessionToken", token);
-                res.header("Location", "/dash.html");
+                res.header("Location", "/admin/dash.html");
```

Comment:
Not sure dash should be on /admin, administrators need to be able to access the page before they've logged in